src/v2/Components/ArtworkFilter/index.tsx

```diff
@@ -130,6 +138,8 @@ export const BaseArtworkFilter: React.FC<
     Filters?: JSX.Element
     offset?: number
     enableCreateAlert?: boolean
+    artistId?: string
+    artistName?: string
```

Comment:
(blocking) Similar to @dimatretyak's comment above, we shouldn't be passing feature-specific props to our generic artwork filter

Reply:
Can we pass an object with needed data?
Something like this:
```
{
  type: "artist",
  name: "artist name",
  id: "artist id",
}
```
Or we should pass all info with viewer? It may cause some problems with viewer types, because we have viewer type for artist, collection, artist series etc. So we'll should check type of viewer and if it is `ArtistArtworkFilter_artist` render `CreateAlertButton`

Reply:
@damassi What do you think? ☝️